_pages/sponsors.html

```diff
@@ -49,6 +49,11 @@ <h2 class="text-center v-pad-bottom">{{ level }}</h2>
         </div>
         <div class="column medium-8">
           <h3 class="partner-name"><a href="{{ sponsor.url_target }}" rel="sponsored">{{ sponsor.name }}</a></h3>
+          {% if sponsor.extra_packages %}
+            {% for package in sponsor.extra_packages %}
+            <p class="partner-package">Extra package {{ package }} donated</p>
```

Comment:
We need to avoid the word "donated" - sponsorship isn't a donation.

Maybe "Additional sponsorship: {{...}}" and make the extra packages a comma-separated list?
I'm trying to think how this will appear for other types of add-on package, eg social event sponsorship (it would read like a company had donated a social event, which would be weird)

I'm trying to imagine how this

Reply:
Sorry for the wrong translation, I'll update the wording. 

Just to be sure I understand correctly, currently we should have something like that :
> Additional sponsoring: x
> Additional sponsoring: y
> ...

And we would like to have :
> Additional sponsoring: x, y
> ...

Is it correct?

Reply:
How about if we don't have the "additional sponsorship" text at all, just a lozenze with "Diversity sponsor", "Social event sponsor" etc?

![sponsor_web](https://github.com/djangocon/2023.djangocon.eu/assets/6770950/b88c3b21-8f24-4f89-8d02-cc673ae8e812)

I think it's fine if they are the same colour, since we don't have any sponsors with more than one sponsoship add-on.

Reply:
I agree, I think it's better like this, I can make the changes tomorrow, I'm not sure I will have time today, I will see